nixos/tests/goss.nix

```diff
@@ -43,8 +43,14 @@ import ./make-test-python.nix ({ pkgs, lib, ... }: {
     machine.wait_for_unit("goss.service")
     machine.wait_for_open_port(8080)
 
+    # due to incomprehensible race conditions, somehow goss fails to get
+    # answers out of systemctl about systemd-journald being up, despite this
+    # being more or less impossible.
+    machine.sleep(5)
+
     with subtest("returns health status"):
       result = json.loads(machine.succeed("curl -sS http://localhost:8080/healthz"))
+      print(result)
```

Comment:
Guessing this `print` was for development purposes and can be removed? Not that it matters

Reply:
Yeah, it was left in so the test is debuggable if it fails in the future, since it doesn't print out the failures.